presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2452,18 +2452,24 @@ public MaterializedViewStatus getMaterializedViewStatus(ConnectorSession session
                                     viewToBaseIndirectMappedColumns);
                         }));
 
+        int missingPartitions = 0;
```

Comment:
Previously we were checking for missing partitions per base table, now we are checking for their sum across all base tables. What is the reason behind this change?

Reply:
I think if there are 0 missing partitions we should consider it as fully materialized- there is the possibility we hit the condition `!dataPredicates.getPredicateDisjuncts().isEmpty()` but filter any missing predicates now.